solvebio/test/test_dataset.py

```diff
@@ -73,3 +73,21 @@ def test_dataset_facets(self):
         field = dataset.fields('status')
         facets = field.facets()
         self.assertTrue(len(facets['facets']) >= 0)
+
+    def test_dataset_metadata(self):
+        dataset = self.client.Dataset.get_by_full_path(
+            self.TEST_DATASET_FULL_PATH)
```

Comment:
this is not obvious (and a bad implementation of unit testing on our part).....but TEST_DATASET_FULL_PATH is an actual real public dataset on SolveBio and we shouldnt modify it.

a cleaner test would be to create a new folder and test the metadata case with that.

folder = self.client.Object.get_or_create_by_full_path('~/test_folder_metadata', object_type='folder')

you can move this test into `test_objects.py`

Reply:
This makes sense. Will do that.